UKHomeOffice/sgmr-service
# COP-3337: Put a new home page on dev

## Description
This branch updates the "homepage" of the service for the new designs. It changes the content/design and moves the notification overview tabs to a new page (called ManageNotifications). 

To test:
- Sign in and go to the home page (http://localhost:8080/reports) to see the new design. If you have any notifications already saved (draft / submutted etc) the numbers on the right should show this.
- Click on the Manage your notifications link under the numbers. This takes you to the new tabs page where you should see all existing notifications. 
- Make a new voyage or delete and existing one. When you navigate back to the homepage the numbers will have updated.

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
